migrations/Version202605191200001101_taoQtiItem.php

```
@@ -0,0 +1,55 @@
+use oat\generis\model\OntologyRdfs;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiItem\model\qti\metadata\guardians\ItemLabelMetadataGuardian;
+
+final class Version202605191200001101_taoQtiItem extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Register service ' . ItemLabelMetadataGuardian::class;
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)